### PR TITLE
Updates on domain delegation

### PR DESCRIPTION
**Pull request by lokst**
Updates the following articles to include the upcoming `Use DNSimple name servers` button and clarify about glue records
- [Setting name servers](https://deploy-preview-1287--dnsimple-support.netlify.app/articles/setting-name-servers/)
- [Delegating a Domain registered with DNSimple to DNSimple](https://deploy-preview-1287--dnsimple-support.netlify.app/articles/delegating-dnsimple-registered/)

## To Do
- [x] Tag @itsalyse for approval
- [x] Tag customer success for approval
- [x] Wait for release date

### CODE REVIEW

File: content/articles/delegating-dnsimple-registered.md [hunk 1]
```diff
@@ -23,18 +23,22 @@ Switching the name servers to DNSimple will cause the domain to resolve using th
...
-1.  Enter the [DNSimple name servers](/articles/dnsimple-nameservers)
+1. Click on <label>Use DNSimple name servers</label> to replace the name servers in the form with [DNSimple name servers](/articles/dnsimple-nameservers). 
+
+    ![Use DNSimple name servers](/files/use-dnsimple-name-servers.png)
+
+1. Click on <label>Use DNSimple name servers</label> in the following confirmation dialog.
+
+    ![Use DNSimple name servers confirmation](/files/use-dnsimple-name-servers-confirmation.png)
+
+1. Alternatively, the [DNSimple name servers](/articles/dnsimple-nameservers) can also be keyed into the form manually
```
Comment (jacegu) [inline content/articles/delegating-dnsimple-registered.md:34]:
```suggestion
1. Alternatively, the [DNSimple name servers](/articles/dnsimple-nameservers) can also be typed into the form manually
```

Comment (jacegu) [reply content/articles/delegating-dnsimple-registered.md:34]: Don't trust the 🇪🇸 for this, but I think _typed_ or _entered_ makes more sense in this context.

File: content/articles/setting-name-servers.md [hunk 2]
```diff
@@ -58,16 +58,20 @@ DNSimple's listing of NS records for the domain will be updated to match the nam
...
+If you are adding a name server that is a child of the domain, glue records are required. In this case, you will be prompted about the missing glue records upon clicking <label>Change Name Servers</label>. The glue records can be added by creating a [name server set](/articles/name-server-sets/#creating-an-account-name-server-set) and clicking on <label>Add a name server set</label> to apply it on the <label>Edit delegation</label> page.
 
 <div class="mb4 aspect-ratio aspect-ratio--16x9 z-0">
   <iframe src="https://www.youtube.com/embed/m_RaPIRNxFs?rel=0&modestbranding=1&cc_load_policy=1&cc_lang_pref=en" class="aspect-ratio--object" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
 </div>
 
+<info>
+The above video also demonstrates adding missing glue records directly from the Edit delegation page. However, this is no longer supported. Glue records can only be added via applying [name server sets](/articles/name-server-sets/#creating-an-account-name-server-set) that contain the necessary glue records.
```
Comment (jacegu) [inline content/articles/setting-name-servers.md:72]: I wonder if we need to remove the video completely (or record a new one) 🤔

**Review by jacegu — APPROVED** [review]
LGTM.

We definitely need @itsalyse on this one 😸

File: content/articles/setting-name-servers.md (hunk 2 above)
Comment (lokst) [reply content/articles/setting-name-servers.md:72]: I think the video is really well-made and most of the video still contains useful information. Perhaps we can leave it up for now and see if this clarifying note is sufficient?

Comment (stephanoodle) [reply content/articles/setting-name-servers.md:72]: I agree that we should leave the video for now.

**Review by stephanoodle — APPROVED** [review]

File: content/articles/delegating-dnsimple-registered.md (hunk 1 above)
Comment (lokst) [reply content/articles/delegating-dnsimple-registered.md]: Thanks for the feedback @jacegu @itsalyse ! I have made the amendment here: https://github.com/dnsimple/dnsimple-support/pull/1287/commits/07d67eb2c257e8166d371d62be591b00fe17456a

**Comment by jacegu** [comment]
@lokst as I was reviewing the project and taking a second look into what support articles needed an update, I wonder if we should add a section to thins one that explains what a "Reserved name server" is, and show a screenshot of how that looks like. WDYT?

**Comment by lokst** [comment]
> @lokst as I was reviewing the project and taking a second look into what support articles needed an update, I wonder if we should add a section to thins one that explains what a "Reserved name server" is, and show a screenshot of how that looks like. WDYT?

@jacegu That makes sense to me! If it's OK, I will open a separate PR to add on those updates

File: content/articles/setting-name-servers.md (hunk 2 above)
Comment (Pemacaa) [reply content/articles/setting-name-servers.md:72]: I also agree

**Review by Pemacaa — APPROVED** [review]

**Review by itsalyse — APPROVED** [review]

**Review by stephanoodle — APPROVED** [review]

**Pull request merged**